ouroboros-network/test/Test/Mux.hs

```diff
@@ -148,6 +156,51 @@ newtype DummyPayload = DummyPayload {
 instance Show DummyPayload where
     show d = printf "DummyPayload %d\n" (BL.length $ unDummyPayload d)
 
+-- |
+-- Generate a byte string of a given size.
+--
+genByteString :: Int64 -> Gen BL.ByteString
+genByteString size = MkGen $ \(QCGen g0) _ ->
+  if size <= 0
+    then BL.empty
```

Comment:
Is the empty special case needed? Isn't it handled by the `gen` general case?

Reply:
That's right.